Macro `__CUDA_PROFILING` doesn't compile anymore

One of the nice features of annotating events in nvprof was based on the macro `__CUDA_PROFILING`, however the code doesn't compile anymore:

```
mpif90 -c -D__FFTW3 -D__parallel -D__SCALAPACK -D__LIBXSMM -D__ACC -D__DBCSR_ACC -D__LIBINT -D__MAX_CONTR=4 -D__LIBXC -D__ELPA -D__PW_CUDA -D__CUDA_PROFILING -O3 -g -fopenmp -mavx2 -mfma -funroll-loops -ffree-form -ffree-line-length-512 -I/home/users/alazzaro/libs/libxsmm/install/include -I/home/users/alazzaro/libs/libint-v2.6.0-cp2k-lmax-7/install/include -I/home/users/alazzaro/libs/libxc-4.3.4/install/include -I/home/users/alazzaro/libs/elpa-2019.05.001/install/include/elpa_openmp-2019.05.001/elpa -I/home/users/alazzaro/libs/elpa-2019.05.001/install/include/elpa_openmp-2019.05.001/modules -I/opt/cray/pe/fftw/3.3.8.5/broadwell/include -D__SHORT_FILE__="\"dbcsr_mm_types.F\"" -I'/cray/css/users/alazzaro/cp2k/cp2k/exts/dbcsr/src/mm/' -I'/cray/css/users/alazzaro/cp2k/cp2k/exts/dbcsr/src' dbcsr_mm_types.F90 
/cray/css/users/alazzaro/cp2k/cp2k/exts/dbcsr/src/acc/acc_stream.cpp: In function 'int acc_stream_create(void**, const char*, int)':
/cray/css/users/alazzaro/cp2k/cp2k/exts/dbcsr/src/acc/acc_stream.cpp:57:24: error: 'custream' was not declared in this scope
   nvtxNameCudaStreamA(*custream, name);
                        ^~~~~~~~
/cray/css/users/alazzaro/cp2k/cp2k/exts/dbcsr/src/acc/acc_stream.cpp:57:24: note: suggested alternative: 'CUstream'
   nvtxNameCudaStreamA(*custream, name);
                        ^~~~~~~~
                        CUstream
```

@shoshijak Is it something you can fix? 
I know that the macro is not in CMake, but it is something we probably want to have it for future performance analysis...

--- a/src/acc/acc_stream.cpp
+++ b/src/acc/acc_stream.cpp
@@ -54,7 +54,7 @@ extern "C" int acc_stream_create(void** stream_p, const char* name, int priority
   if (acc_error_check(ACC(GetLastError)())) return -1;
 
 #ifdef __CUDA_PROFILING
-  nvtxNameCudaStreamA(*custream, name);
+  nvtxNameCudaStreamA(*acc_stream, name);
 #endif
 
     return 0;

--- a/src/acc/cuda/dbcsr_cuda_nvtx_cu.cu
+++ b/src/acc/cuda/dbcsr_cuda_nvtx_cu.cu
@@ -31,7 +31,7 @@ const uint32_t colormap[] = { 0xFFFFFF00,  // Yellow
 //==============================================================================
 extern "C" int cuda_nvtx_range_push_cu(const char* message) {
 
-    //assembling event attribute
+    // assembling event attribute
     nvtxEventAttributes_t eventAttrib = {0};
     eventAttrib.version = NVTX_VERSION;
     eventAttrib.size = NVTX_EVENT_ATTRIB_STRUCT_SIZE;
@@ -45,7 +45,7 @@ extern "C" int cuda_nvtx_range_push_cu(const char* message) {
     eventAttrib.colorType = NVTX_COLOR_ARGB;
     eventAttrib.color = colormap[hash%14];
 
-    //these field could be fild with useful stuff
+    // these field could be filled with useful stuff
     eventAttrib.payloadType = NVTX_PAYLOAD_TYPE_INT64;
     eventAttrib.payload.llValue = 123;
     eventAttrib.category = 42;

